lp-steam/templates/csgo/server.j2

```diff
@@ -22,14 +22,34 @@ sv_lan 1
 host_players_show 2
 host_info_show 2
 
+game_type 0 
+game_mode 1
+
 {% if item.value.gotv | default(false)  %}
+tv_port "{{ item.value.gameport + 5 }}" 
 tv_enable "1"
 tv_title "{{ lp.name }} GOTV {{ steam_server_id }}.{{ item.key }}"
 tv_name "{{ lp.name }} GOTV {{ steam_server_id }}.{{ item.key }}"
 tv_password "{{ csgo_gotv_password | default('stream') }}"
 tv_maxclients "{{ csgo_gotv_maxclients | default(4) }}"
 tv_snapshotrate "{{ csgo_gotv_snaprate | default(64) }}"
-tv_delaymapchange 1
+tv_delaymapchange "1" 
```

Comment:
Lots of extra whitespace on these lines, and most of them are defaults:
```
tv_allow_static_shots "1" // default
tv_autoretry "1" // default
tv_chatgroupsize "0" // default
tv_chattimelimit "8" // default
tv_debug "0" // default
tv_deltacache "2" // default
tv_transmitall "1" // default
```

These will be removed, but can be enforced in an external exec file.